### PR TITLE
Add changelog entries for 11.1.2

### PR DESCRIPTION
**Pull request by onurctirtir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+### citus v11.1.2 (September 30, 2022) ###
+
+* Disallows having `ON DELETE/UPDATE SET DEFAULT` actions on columns that
+  default to sequences
+
+* Fixes a bug that might cause inserting incorrect `DEFAULT` values when
+  applying foreign key actions
+
+* Fixes a performance issue related to shard-moves by creating replica
```
Comment (onurctirtir) [inline CHANGELOG.md:9]: https://github.com/citusdata/citus/pull/6367/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+  identities before copying shards
+
+* Improves logging during shard-splits and resource cleanup
+
+* Makes sure to reuse connections for shard-splits and logical replication
+
+* Makes sure to try dropping replication slots a few more times after a failure
```
Comment (onurctirtir) [inline CHANGELOG.md:16]: https://github.com/citusdata/citus/pull/6359

**Review by onurctirtir — COMMENTED** [review]
Left links to the PRs for which I needed to modify related changelog entries a bit

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+* Fixes a bug that might cause inserting incorrect `DEFAULT` values when
+  applying foreign key actions
+
+* Fixes a performance issue related to shard-moves by creating replica
+  identities before copying shards
+
+* Improves logging during shard-splits and resource cleanup
+
+* Makes sure to reuse connections for shard-splits and logical replication
```
Comment (onurctirtir) [inline CHANGELOG.md:14]: and this https://github.com/citusdata/citus/pull/6314

**Review by hanefi — APPROVED** [review]

**Pull request merged**